webapp-django/crashstats/crashstats/static/crashstats/js/socorro/crontabber_state.js

```diff
@@ -1,67 +1,9 @@
 /* global _, d3 */
 
-var margin = { top: 1, right: 20, bottom: 6, left: 10 };
```

Comment:
I thought about redoing this file without d3, but it wasn't super valuable and I figured someone could do that later.